cloudstate/evensourced_servicer.py

```diff
@@ -0,0 +1,121 @@
+from dataclasses import dataclass
+from typing import List
+
+from google.protobuf.any_pb2 import Any
+
+from cloudstate.entity_pb2 import Command
+from cloudstate.event_sourced_context import SnapshotContext, EventSourcedCommandContext, EventContext
+from cloudstate.event_sourced_entity import EventSourcedEntity, EventSourcedHandler
+from cloudstate.event_sourced_pb2 import EventSourcedInit, EventSourcedSnapshot, EventSourcedEvent, EventSourcedReply, \
+    EventSourcedStreamOut
+from cloudstate.event_sourced_pb2_grpc import EventSourcedServicer
+from pprint import pprint
+from google.protobuf import symbol_database as _symbol_database
+import logging
+
+_sym_db = _symbol_database.Default()
+
+TYPE_URL_PREFIX = 'type.googleapis.com/'
+
+
+def get_payload(command):
+    command_type: str = command.payload.type_url
+    if command_type.startswith(TYPE_URL_PREFIX):
+        command_type = command_type[len(TYPE_URL_PREFIX):]
+    command_class = _sym_db.GetSymbol(command_type)
+    cmd = command_class()
+    cmd.ParseFromString(command.payload.value)
+    return cmd
+
+
+def pack(event):
+    any = Any()
+    any.Pack(event)
+    return any
+
+
+class CloudStateEventSourcedServicer(EventSourcedServicer):
```

Comment:
How about we rename it to CloudstateEventSourcedService?

Reply:
yeah, makes sense since LB renamed it tha way 👍